Source/RapyutaSimulationPlugins/Private/Core/RRGameState.cpp

```diff
@@ -89,6 +92,19 @@ void ARRGameState::StartSim()
     }
 }
 
+void ARRGameState::RegisterSpawnableBPClasses()

```

Comment:
Shouldn't these logic be part of GameMode since GameMode has  SimulationState.

Reply:
yes, it should be server-only, so moved to SimulationState & invoked by ROS2GameMode